src/game.cpp

```diff
@@ -7685,8 +7685,12 @@ void game::examine( const tripoint &examp )
                 return;
             }
         }
+        if( mon != nullptr && mon->has_flag( MF_MILKABLE ) ) {
+            add_msg( m_info, _( "This is a milkable cow." ) );
```

Comment:
Actually it's a milkable monster, nothing here checks that it's a cow. You might want to add the monster type name instead of "cow".